firefox_puppeteer/api/places.py

```diff
@@ -0,0 +1,156 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette.errors import TimeoutException
+
+from ..base import BaseLib
+
+
+class Places(BaseLib):
+    """Low-level access to several bookmark and history related actions."""
+
+    bookmark_folders = {'placesRoot': 1,
+                        'bookmarksMenuFolder': 2,
+                        'toolbarFolder': 3,
+                        'tagsFolder': 4,
+                        'unfiledBookmarksFolder': 5,
+                        }
```

Comment:
I mean one class variable that is a named tuple. Dictionaries are good for mapping arbitrary values to arbitrary things, this is more like an enumeration, so I'd like to be able to refer to something like places.bookmark_folders.placesRoot in code.